admin-manual/installation-setup/integrations/duracloud-setup.rst

```diff
@@ -1,226 +1,101 @@
-and may also be helpful for those using the hosted service. This guide assumes
-that you have already installed Archivematica and DuraCloud, or have arranged
-for a hosted instance through Artefactual Systems/DuraSpace. Hosted clients will
-have steps 1 and 2 completed for them.
+Setting up the DuraCloud integration requires configuration of Archivematica's
+Storage Service. It is recommended that you read the :ref:`Storage Service
+documentation <storageservice:administrators>` before configuring DuraCloud.
 
-For installation instructions please see:
+This guide can be used by those who wish to install Archivematica and/or
+DuraCloud locally and run the systems themselves and may also be helpful for
+those who subscribe to a hosted service. `ArchivesDirect`_ is an Archivematica
+in DuraCloud hosting plan offered by Artefactual Systems and DuraSpace.
 
-* :ref:`Archivematica installation <installation>`
-* `DuraCloud documentation`_
+*On this page*
 
-Step 1: Configure DuraCloud
----------------------------
+* :ref:`Configure DuraCloud <duracloud-config>`
+* :ref:`Configure the Archivematica Storage Service <duracloud-storage-service>`
 
-If you plan to use DuraCloud for transfer source, processing and AIP/DIP storage,
-it is recommended that your DuraCloud instance should be configured to have the
-following spaces:
+.. _duracloud-config:
 
-* aip-store : This is the space where completed AIPs will be stored
+Configure DuraCloud
+-------------------
 
-* dip-store : Space where completed DIPs are stored
+Your DuraCloud instance must have at least two users:
 
-* transfer-source : Space to sync digital objects for transfer into
-  Archivematica. You may also create multiple spaces with other names for this
-  purpose.
+* A **Client user** who has *read* permissions for the AIP storage and DIP
+  storage locations.
 
-* transfer-backlog : Space used by Archivematica for material sent to
-  :ref:`backlog <backlog>`.
+* An **Archivematica user** who has *read/write* permissions for the AIP
+  storage, DIP storage, and transfer backlog locations.
 
-Note that you can pick and choose what spaces are needed in DuraCloud; for
-example, you may choose to use a local server for transfer source but send
-your AIPs and DIPs for storage in DuraCloud. You need only create the relevant
-spaces in DuraCloud for your particular configuration.
+DuraCloud can be used as a storage space for the following :ref:`purposes
+<storageservice:location-purposes>` within Archivematica:
 
-Two DuraCloud users should be created:
+* **AIP storage**: A DuraCloud space where completed AIPs are placed for
+  long-term storage.
+* **DIP storage**: A DuraCloud space where DIPs can be stored. Note that it is
+  not necessary to store DIPs if you are sending them directly to an access
+  system.
+* **Transfer backlog**: A DuraCloud space used by Archivematica for material
+  sent to the :ref:`backlog <backlog>` during processing.
 
```

Comment:
I wonder if we can note that this is a recommendation, i.e. there are no technical reasons all three couldn't be `<a bucket>`. In doing this I feel it helps folk with the logic of what they're doing, as well as the logic of something that might go wrong e.g. if accidentally set up to be one bucket, then that's because it's all feasible. And we could add a lint of digipres and records and info mgmt too -- something along the lines of, management of these package types are best done separated into multiple bucket types dedicated to a storage service's purpose.

Reply:
Thanks, I've included a note to that effect.